.gitignore

```diff
@@ -163,4 +163,6 @@ tags
 *.lock
 
 # DS_Store (MacOS)
-.DS_Store
\ No newline at end of file
+.DS_Store
```

Comment:
@bglick13 to get the PR ready, let's remove things like `hub`. 
We can leave `*.mp4` if we add a pipeline, but add a comment saying its for rl so people know.